content/en/docs/Tracing/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Tracing
+description: Tracing Kyverno engine admission requests processing
+weight: 65
+---
+
+## Introduction
+
+Tracing is a method of tracking application requests as they are processed by the application software. When a software is instrumented it produces traces, traces are made of spans hierarchically organised to form a trace. Spans are defined by a start and end time, eventually a parent span, and a number of properties that define the span characteristics (client spans, server spans, etc...).
+
+Tracing is not limited to a single application, as the tracing context can be transmitted on the wire it is possible to collect spans from multiple applications and reconcile them in a single trace.
+
+In the context of Kyverno, requests are usually sent by the Kubernetes API server to the Kyverno service during the admission phase. Kyverno receives and processes admission requests according to the configured policies. Every step in the admission pipeline and during the engine policy processing will produce spans. All clients (Kubernetes client, registry client and cosign client) have also been instrument to produce client spans and transmit the tracing context on the wire.
+
+## Trace example
+
+Below is a trace for a validating admission request.
+
+<p align="center"><img src="https://raw.githubusercontent.com/kyverno/website/main/content/en/docs/Tracing/assets/trace-example-1.png" height="300px"/></p>
```

Comment:
You don't need to go to this much trouble to reference an image and, anyway, the link is broken. See an example in the Installation page.

Reply:
Fixed

Reply:
We may want to fix the same thing in monitoring docs